internal/archive/zip/zip.go

```diff
@@ -52,10 +52,10 @@ func (w *Writer) Add(src, dst string) (count int, length int, err error) {
 	}
 
 	log.Debug().Str("name", src).Msg("Adding to archive")
-	target := path.Join(dst, finfo.Name())
+	target := filepath.Join(dst, finfo.Name())
```

Comment:
Hm, the docs for https://pkg.go.dev/archive/zip#Writer.Create say that only forward slashes are allowed though 🤔 

Looking at its implementation, it does check for directories via
```
if strings.HasSuffix(fh.Name, "/") {
```

which would not be compatible with your changes.

Reply:
Also see the [zip file format specification](https://pkware.cachefly.net/webdocs/casestudies/APPNOTE.TXT) under `4.4.17 file name: (Variable)`:
```
The name of the file, with optional relative path.
       The path stored MUST NOT contain a drive or
       device letter, or a leading slash.  All slashes
       MUST be forward slashes '/' as opposed to
       backwards slashes '\' for compatibility with Amiga
       and UNIX file systems etc.  If input came from standard
       input, there is no file name field.
```

Reply:
hmm, I did a test to verify it https://go.dev/play/p/hpJx_qFP_RQ No problem on windows platform 🤔

Reply:
Updated 😆